keith-vscode/src/util.ts

```diff
@@ -15,56 +15,58 @@
-* @param uri URI of the document where the edit should be applied.
-* @param text The text to insert in the document.
-* @param position The position in the document where the text should be inserted.
-*/
+ * Applys a workspaceedit to the document defined by {@code uri}, at the position {@code position} with the given {@code text}.
+ * @param uri URI of the document where the edit should be applied.
+ * @param text The text to insert in the document.
+ * @param position The position in the document where the text should be inserted.
+ */
 export async function handleWorkSpaceEdit(uri: string, text: string, position: vscode.Position): Promise<void> {
     // get the desired editor and document
-    const editor = vscode.window.visibleTextEditors.find(visibleEditor => visibleEditor.document.uri.toString() === uri);
-    const textDocument = editor?.document;
+    const editor = vscode.window.visibleTextEditors.find(
+        (visibleEditor) => visibleEditor.document.uri.toString() === uri
+    )
+    const textDocument = editor?.document
     if (!textDocument) {
         vscode.window.showErrorMessage(
             `Server requested a text edit but the requested uri was not found among the known documents: ${uri}`
-        );
-        return;
+        )
+        return
     }
     // create the insert workspaceedit
-    const workSpaceEdit = new vscode.WorkspaceEdit();
-    const edits: vscode.TextEdit[] = [vscode.TextEdit.insert(position, text)];
-    workSpaceEdit.set(textDocument.uri, edits);
+    const workSpaceEdit = new vscode.WorkspaceEdit()
+    const edits: vscode.TextEdit[] = [vscode.TextEdit.insert(position, text)]
+    workSpaceEdit.set(textDocument.uri, edits)
 
     // Apply and save the edit. Report possible failures.
-    const edited = await vscode.workspace.applyEdit(workSpaceEdit);
+    const edited = await vscode.workspace.applyEdit(workSpaceEdit)
     if (!edited) {
-        vscode.window.showErrorMessage("Workspace edit could not be applied!");
-        return;
+        vscode.window.showErrorMessage('Workspace edit could not be applied!')
+        return
     }
 
     if (editor) {
         // reveal the range of the inserted text
-        const endPos = textDocument.positionAt(textDocument.offsetAt(position) + text.length);
-        editor.selection = new vscode.Selection(position, endPos);
-        editor.revealRange(new vscode.Range(position, endPos));
+        const endPos = textDocument.positionAt(textDocument.offsetAt(position) + text.length)
+        editor.selection = new vscode.Selection(position, endPos)
+        editor.revealRange(new vscode.Range(position, endPos))
     }
 
-    const saved = await textDocument.save();
+    const saved = await textDocument.save()
     if (!saved) {
-        console.error(`TextDocument ${textDocument.uri} could not be saved!`);
-        return;
+        // eslint-disable-next-line no-console
+        console.error(`TextDocument ${textDocument.uri} could not be saved!`)
```

Comment:
Please replace the console.error with vscode.window.showErrorMessage

Reply:
See #62